m2e-configurator/net.revelc.code.formatter/src/net/revelc/code/formatter/connector/internal/FormatterProjectConfigurator.java

```diff
@@ -125,14 +126,20 @@ private InputStream readConfigFile(Formatter formatter, ProjectConfigurationRequ
             javaConfigFile = formatter.getDefaultPath();
         }
 
-        IFile cfgFile = request.getProject().getFile(javaConfigFile);
-
-        if (!cfgFile.exists()) {
-            throw new CoreException(
-                    new Status(IStatus.CANCEL, FormatterCore.PLUGIN_ID, "Configuration file not found!"));
+        File cfgFile = new File(javaConfigFile);
+        if (!cfgFile.isAbsolute()) {
```

Comment:
I think all this is doing different is checking to see if an absolute path exists.  If it does, it does the same as it always did.  If it does not, then it performs the additional check.  So worst case this is just adding something, best case it does as stated.  As I don't use the m2 configuration not sure how this completely works.  I did try to go to the core plugin and implement something similar but that didn't really do anything.  I did however get full multi module support so format file can be in parent and no matter how deep the maven build is, it will recursively search.  Also of note, plexus utils got much smarter so old lost logic of jar not working now works.  So we should be relatively close to release at this point.  As for this change, is this something you can test based on what user documented?